lib/build_targets/package.dart

```diff
@@ -295,6 +295,8 @@ class NativeTpk extends TizenPackage {
       'capi-appfw-app-common',
       'capi-appfw-application',
       'dlog',
+      'elementary',
```

Comment:
Is `elementary` for future use?

Reply:
Since Elemental.h is explicitly included in elm_flutter_view.h, I added it. should I except it?

Reply:
Not really. It might be useful in the future.